lib/src/cross_platform/native.dart

```diff
@@ -19,19 +19,36 @@ int _sizeOfEntity(FileSystemEntity entity) {
   return stat.size;
 }
 
-void copyDir(String from, String to) {
-  _copyDirImpl(Directory(from), Directory(to));
+String copyDir(String from, String to) {
+  final destDir = Directory(join(to, 'ditto-export-${DateTime.now().millisecondsSinceEpoch}'));
```

Comment:
The name `copyDir` implies that it's a generic copy utility - maybe we could name it `copyToDittoExportDir` or something to make it clear it's doing some sort of extra logic?

Reply:
this is fixed as I'm moving over to using zip file instead.